contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -428,6 +439,8 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
   }
 
   event AdminPrivilegeConfigSet(address indexed admin, bytes privilegeConfig);
+  event BillingConfigOverridden(uint256 indexed id);
```

Comment:
can we add the BillingConfigOverride to this event?

Reply:
Do you mean renaming this event from `BillingConfigOverridden ` to `BillingConfigOverride `?

Reply:
oh sorry I meant like `event BillingConfigOverridden(uint256 indexed id, BillingConfigOverride override);`

Reply:
Got it!